paddle3d/thirdparty/kitti_object_eval_python/eval.py

```diff
@@ -509,7 +509,7 @@ def eval_class(gt_annos,
     split_parts = get_split_parts(num_examples, num_parts)
 
     rets = calculate_iou_partly(
-        dt_annos, gt_annos, metric, num_parts, z_axis=z_axis, z_center=z_center)
+        gt_annos, dt_annos, metric, num_parts, z_axis=z_axis, z_center=z_center)
```

Comment:
Simply reversing the order doesn't give correct results. See my comments for details.

Reply:
Got it